hedera-node/src/main/java/com/hedera/services/stream/RecordStreamType.java

```diff
@@ -40,14 +40,24 @@
 	 */
 	public static final String RECORD_SIG_EXTENSION = "rcd_sig";
 	/**
-	 * Header which is written in the beginning of a stream file, before writing the Object Stream Version.
-	 * the ints in fileHeader denote: version 5, hapiProtoVersion: 0.11.0
+	 * The {@link com.swirlds.common.stream.TimestampStreamFileWriter} writes this header at the beginning of a
+	 * stream file (*.rcd), immediately before its internal stream version.
+	 *
+	 * The four ints in ths header denote, in order:
+	 * <ol>
+	 *     <li>The Services record stream version {@literal X}</li>
+	 *     <li>The HAPI protobuf major version</li>
+	 *     <li>The HAPI protobuf minor version</li>
+	 *     <li>The HAPI protobuf patch version</li>
+	 * </ol>
+	 * So the header below denotes a V5 record stream that uses semantic version 0.23.0 of the HAPI protobufs.
 	 */
-	private static final int[] RECORD_FILE_HEADER = new int[] { 5, 0, 11, 0 };
+	private static final int[] RECORD_FILE_HEADER = new int[] { 5, 0, 23, 0 };
```

Comment:
Can we figure out a way to automate this in the next release?

Reply:
Hmm good question...let me try.

Reply:
Will be a bit delicate since test assets also depend on this value tho...will ponder for a separate issue. 🤔